content/es/docs/contribute/start.md

```diff
@@ -18,7 +18,7 @@ Si estas buscando información sobre cómo comenzar a contribuir a los repositor
 
 ## Lo básico sobre nuestra documentación
 
-La documentación de Kuberentes esta escrita usando Markdown, procesada y
+La documentación de Kubernetes esta escrita usando Markdown, procesada y
```

Comment:
I won't touch other languages unless I speak that language.
Who knows if in Spanish "kubernetes" has a different spelling?

Reply:
There are many string "Kubernetes" in this file, "Kubernetes" is a proper noun, so I fix the wrong string.

Eg.
## Envía un caso de estudio

Un caso de estudio destaca como organizaciones están usando Kubernetes para resolver problemas del mundo real. Estos se escriben en colaboración con el equipo de marketing de Kubernetes que está dirigido por la {{< glossary_tooltip text="CNCF" term_id="cncf" >}}.

Reply:
https://kubernetes.io/docs/contribute/localization/#help-an-existing-localization